refinery/static/source/js/refinery/contents.js

```diff
@@ -11,161 +11,162 @@
 
   var dataQueryString = undefined;
 
-  var currentStudyUuid = externalStudyUuid;
-  var currentStudyId = externalStudyId;
-  var currentAssayUuid = externalAssayUuid;
-  var currentAssayId = externalAssayId;
+  var currentStudyUuid = '';
+  var currentStudyId = '';
+  var currentAssayUuid = '';
+  var currentAssayId = '';
```

Comment:
I'm not sure you want to get into it now, but I think currentStudyId and currentAssayUuid are unused, and the others could go inside the scope of the ready. Or leave it for now.

Reply:
That's part of the next phase.